### PR TITLE
add e2e test that runs on latest raw release in cron jobs only

### PR DESCRIPTION
**Pull request by jakemac53**
cc @kevmoo @matanlurey for fyi, I think this should work... :D

### CODE REVIEW

**Comment by kevmoo** [comment]
Once this lands I'll bump the cron run frequency to daily – and we'll see.

Maybe add "api" to the list, too? Then we can hit it manually?

File: build_resolvers/pubspec.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ author: Dart Team <misc@dartlang.org>
 homepage: https://github.com/dart-lang/build/tree/master/build_resolvers
 
 environment:
-  sdk: '>=2.0.0-dev.23 <2.0.0'
+  sdk: '>=2.0.0-dev.32 <2.0.0'
```
Comment (natebosch) [inline build_resolvers/pubspec.yaml]: why so high? I thought `orElse` was added a few releases ago?

Comment (jakemac53) [reply build_resolvers/pubspec.yaml]: @kevmoo said it just got released, I took his word for it because digging for it is more effort than its worth.

Comment (matanlurey) [reply build_resolvers/pubspec.yaml]: I'm supportive of this, less variance in having to figure out what dev SDK works and doesn't (especially given most dev users are using brew/automatically updating anyway).

**Comment by jakemac53** [comment]
merging on red as we are red for other issues (stack traces getting broken)

**Comment by kevmoo** [comment]
It worked!

https://travis-ci.org/dart-lang/build/builds/348382639

Used the trigger build option – scroll to the bottom to see the cron item

**Comment by kevmoo** [comment]
Just updated the cron job to run daily...

This is where we could wire up the Chat webhook 😄

**Comment by jakemac53** [comment]
nice, will fix the redness shortly so we can see it actually run but ya looks like its working!

**Pull request merged**